OpenRA.Mods.Common/Projectiles/GravityBomb.cs

```diff
@@ -17,6 +17,7 @@
 
 namespace OpenRA.Mods.Common.Projectiles
 {
+	[Desc("Projectile with flexible physics customisation.")]
```

Comment:
"flexible physics customisation" could be a bit more descriptive imo. Something along the lines of "Projectiles with customizable movement and acceleration vector"?

Reply:
I'm going with "Projectile with customisable acceleration vector"